Index: .idea/modules.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/modules.xml b/.idea/modules.xml
new file mode 100644
--- /dev/null	(date 1698442769061)
+++ b/.idea/modules.xml	(date 1698442769061)
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="ProjectModuleManager">
+    <modules>
+      <module fileurl="file://$PROJECT_DIR$/.idea/Catalyst.iml" filepath="$PROJECT_DIR$/.idea/Catalyst.iml" />
+    </modules>
+  </component>
+</project>
\ No newline at end of file
Index: config/database.js
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>const mongoose = require(\"mongoose\")\r\nconst uri = process.env.CONNECTION_STRING;\r\n\r\nconst connection = async () => {\r\n    return mongoose.connect(uri, { useNewUrlParser: true, useUnifiedTopology: true })\r\n        .then(() => {\r\n            console.log(\"Connected to MongoDB database successfully!\");\r\n        }).catch((err) => {\r\n            console.log(\"MongoDB Error: \", err);\r\n        })\r\n}\r\n\r\nmodule.exports = {\r\n    connection,\r\n    mongoose,\r\n    connect: () => {\r\n        mongoose.Promise = Promise;\r\n        mongoose.connect(uri);\r\n    },\r\n    disconnect: done => {\r\n        mongoose.disconnect(done);\r\n    }\r\n};
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/config/database.js b/config/database.js
--- a/config/database.js	(revision 548adea336e6974051b207430e5ecddbca54f34c)
+++ b/config/database.js	(date 1698442769077)
@@ -1,5 +1,6 @@
 const mongoose = require("mongoose")
-const uri = process.env.CONNECTION_STRING;
+const uri = 'mongodb+srv://Catalyst:Catalyst-Team@cluster0.dxxolsb.mongodb.net/Catalyst';
+
 
 const connection = async () => {
     return mongoose.connect(uri, { useNewUrlParser: true, useUnifiedTopology: true })
Index: .idea/vcs.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/vcs.xml b/.idea/vcs.xml
new file mode 100644
--- /dev/null	(date 1698442769054)
+++ b/.idea/vcs.xml	(date 1698442769054)
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="VcsDirectoryMappings">
+    <mapping directory="" vcs="Git" />
+  </component>
+</project>
\ No newline at end of file
Index: .idea/.gitignore
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/.gitignore b/.idea/.gitignore
new file mode 100644
--- /dev/null	(date 1698442769047)
+++ b/.idea/.gitignore	(date 1698442769047)
@@ -0,0 +1,5 @@
+# Default ignored files
+/shelf/
+/workspace.xml
+# Editor-based HTTP Client requests
+/httpRequests/
Index: .idea/Catalyst.iml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/Catalyst.iml b/.idea/Catalyst.iml
new file mode 100644
--- /dev/null	(date 1698442769037)
+++ b/.idea/Catalyst.iml	(date 1698442769037)
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<module type="WEB_MODULE" version="4">
+  <component name="NewModuleRootManager">
+    <content url="file://$MODULE_DIR$">
+      <excludeFolder url="file://$MODULE_DIR$/.tmp" />
+      <excludeFolder url="file://$MODULE_DIR$/temp" />
+      <excludeFolder url="file://$MODULE_DIR$/tmp" />
+    </content>
+    <orderEntry type="inheritedJdk" />
+    <orderEntry type="sourceFolder" forTests="false" />
+  </component>
+</module>
\ No newline at end of file
